### PR TITLE
Fix unicode in ical

### PR DESCRIPTION
**Pull request by mruwek**
Connected to #21

### CODE REVIEW

File: test/IcalCommandTest.py [hunk 1]
```diff
@@ -80,5 +80,31 @@ def test_help_python32(self):
...
         self.assertEqual(self.errors, "icalendar is not supported in this Python version.\n")
 
+class IcalCommandUnicodeTest(CommandTest):
+    def setUp(self):
+        super(IcalCommandUnicodeTest, self).setUp()
+        self.todolist = load_file_to_todolist("test/data/ListCommandUnicodeTest.txt")
+
+    @unittest.skipIf(IS_PYTHON_32, "icalendar is not supported for Python 3.2")
+    def test_ical_unicode(self):
+        def replace_ical_tags(p_text):
```
Comment (bram85) [inline test/IcalCommandTest.py]: Please move this function outside the class, and call it from here and from `IcalCommandTest.test_ical()`. This reduces code duplication.

File: test/IcalCommandTest.py [hunk 2]
```diff
@@ -80,5 +80,31 @@ def test_help_python32(self):
...
+            # replace identifiers with dots, since they're random.
+            result = re.sub(r'\bical:....\b', 'ical:....', p_text)
+            result = re.sub(r'\bUID:....\b', 'UID:....', result)
+
+            return result
+
+        command = IcalCommand([""], self.todolist, self.out, self.error)
+        command.execute()
+
+#        self.assertTrue(self.todolist.is_dirty())
```
Comment (bram85) [inline test/IcalCommandTest.py]: Should be `self.assertFalse(self.todolist.is_dirty())`

Comment (mruwek) [inline test/IcalCommandTest.py]: Of course it should. I've commented it out to test something else, and then forgotten about it. Sorry.

Comment (mruwek) [inline test/IcalCommandTest.py]: Wait, shouldn't it be rather `self.assertTrue(self.todolist.is_dirty())`? - ical adds `ical:` tag here after all.

File: test/IcalCommandTest.py (hunk 1 above)
Comment (mruwek) [inline test/IcalCommandTest.py]: Done.

File: test/IcalCommandTest.py (hunk 2 above)
Comment (bram85) [inline test/IcalCommandTest.py]: Good point.

**Pull request merged**